.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,27 @@
+### JIRA link
```

Comment:
I don't think this applies to an application such as this java client. I would use this one:

https://github.com/hmcts/idam-java-client/blob/master/.github/PULL_REQUEST_TEMPLATE.md

Reply:
you shouldn't need it really, there's a global one applied at the org 

https://github.com/hmcts/.github/blob/master/PULL_REQUEST_TEMPLATE.md